docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+Note that the script watches the files and recompiles on change, so you don't have to "refresh" anything aside from hitting save.
+
+2. When iterating, the Aardvark renderer does *not* have to be restarted to reflect gadget updates but you *do* have to instantiate a new gadget every time you want to see the latest compiled changes (and probably want to discard that gadget after testing).
+This is what my workflow typically looks like - IDE, test in SteamVR, back to IDE:
+
+![workflow.gif](./media/workflow.gif)
+
+3. For debugging, remember you can use Chrome's devtools on the Aardvark Server (http://localhost:8042/), as well as the Gadget Monitor (http://localhost:23842/gadgets/aardvark_monitor/index.html). You can also debug your gadget from VS Code, provided you've installed the Debugger for Chrome extension and answered "yes" to the `Do you want to debug with VS Code?` question when [instantiating your gadget](https://aardvarkxr.github.io/aardvark/getting_started/). 
+
+
+# Step 2: _Time of the Season_ - Adding a time/date panel
+
+Alright, now that we've got a dev environment and a nice workflow, we can jump into the code. 
+
+You can find and download the **necessary media assets** we'll be using [here](https://github.com/MichaelHazani/av-watch-gadget/tree/master/PROJECT_ASSETS). You'll notice that repo is the finished project, but I highly recommend starting from scratch and working your way through the tutorial). 
+
+Let's start by [instantiating a new gadget](https://aardvarkxr.github.io/aardvark/getting_started/). Give it any name you want, and select the defaults for everything (see the [previous tutorial](https://aardvarkxr.github.io/aardvark/getting_started/) for a recap if you'd like). 
+
+We'll be working primarily in `main.tsx` and `styles.css` located in the `src/` folder, which (after running `npm start` as described in the previous tutorial) get recompiled to the `dist/` folder after every save. This is an important fact to keep in mind; for one thing, you'll want to place whatever 3D models and various media files you'll be using in the `dist` folder since that'll be the root of your local references - where `main.tsx` will be solving paths from.
+
+Now, let's shave down the default scene graph created in `main.tsx` to a minimum, so that it only contains the bare bones:
+
+```tsx
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import bind from 'bind-decorator';
+
+import { AvGadget, AvTransform, AvPanel, AvGrabbable, 
+HighlightType, GrabResponse, AvSphereHandle, 
+AvTransformControl, AvPanelAnchor, AvModel } from '@aardvarkxr/aardvark-react';
+
+import { EndpointAddr, AvGrabEvent, endpointAddrToString } from '@aardvarkxr/aardvark-shared';
+
+interface WatchState {}
+
+class WatchGadget extends React.Component<{}, WatchState>
+{
+	constructor(props: any) {
+	super(props);
+	}
+
+	public render() {
+		let scale = 0.2;
+
+return (
+	<div>
+		<AvGrabbable dropOnHooks={true}>
```

Comment:
You might want to use AvStandardGrabbable here. It takes a handle model url and deals with some of the fiddly bits for you. See the white board for an example. It's just newer than the create script so it's not used there yet.

Reply:
thanks! pinged you on slack about that